rotkehlchen/api/v1/schemas.py

```diff
@@ -1656,6 +1649,14 @@ class ExternalServicesResourceDeleteSchema(Schema):
     services = fields.List(SerializableEnumField(enum_class=ExternalService), required=True)
 
 
+class MoneriumOAuthCredentialsSchema(Schema):
+    access_token = NonEmptyStringField(required=True)
+    refresh_token = NonEmptyStringField(required=True)
+    expires_in = fields.Integer(required=True, validate=validate.Range(min=1))
+    client_id = NonEmptyStringField(required=True)
+    token_type = NonEmptyStringField(load_default='Bearer')
```

Comment:
as I said in the docs keep only what you need and also add a docstring here explaining (same as you do in the docs -- what each attribute here does)